### ISSUE TITLE
PIM3 API: Multi reference property value can't be None

### ISSUE THREAD

**Issue by bastiaanbeijer** [KE-works/pykechain#468]
Script failed when the reference value is None.

```
File "/workspace/src/services/import_harness_junctions.py", line 87, in main
    existing_harness_junction_references = ewis_interconnection_location_part.property("Harness junctions").value
  File "/usr/local/lib/python3.6/site-packages/pykechain/models/property2_multi_reference.py", line 69, in value
    category=None))
  File "/usr/local/lib/python3.6/site-packages/pykechain/client.py", line 646, in parts
    raise NotFoundError("Could not retrieve parts: {}".format(response.content))

pykechain.exceptions.NotFoundError: Could not retrieve parts: 

Traceback (most recent call last):
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/generics.py", line 152, in filter_queryset
    queryset = backend().filter_queryset(self.request, queryset, self)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/dry_rest_permissions/generics.py", line 45, in filter_queryset
    return self.filter_list_queryset(request, queryset, view)
  File "/opt/ke-chain/eofw-dev/kechain2/gcore/gpim/api/part/filters_base.py", line 27, in filter_list_queryset
    raise SuspiciousOperation("Part API needs either scope_id, activity_id, parent_id, "
django.core.exceptions.SuspiciousOperation: Part API needs either scope_id, activity_id, parent_id, model_id or (part) id filter.
", "detail": null, "results": [{"traceback": "Traceback (most recent call last):
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/mixins.py", line 40, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/rest_framework/generics.py", line 152, in filter_queryset
    queryset = backend().filter_queryset(self.request, queryset, self)
  File "/opt/ke-chain/eofw-dev/_venv_py34/lib/python3.5/site-packages/dry_rest_permissions/generics.py", line 45, in filter_queryset
    return self.filter_list_queryset(request, queryset, view)
  File "/opt/ke-chain/eofw-dev/kechain2/gcore/gpim/api/part/filters_base.py", line 27, in filter_list_queryset
    raise SuspiciousOperation("Part API needs either scope_id, activity_id, parent_id, "
django.core.exceptions.SuspiciousOperation: Part API needs either scope_id, activity_id, parent_id, model_id or (part) id filter.
", "detail": null, "msg": "Part API needs either scope_id, activity_id, parent_id, model_id or (part) id filter."}], "msg": "Part API needs either scope_id, activity_id, parent_id, model_id or (part) id filter."}'
```

**Comment by jberends** [comment]
must add the 'id' of the part in the PUT request. Need to check with @oikonomou on the precise API call.

**Comment by jberends** [comment]
not in 2.5, PIM2 only iy seems

**Comment by raduiordache** [comment]
@bastiaanbeijer - can you check if this is still valid in PIM3?

**Comment by JelleBoersma** [comment]
Resolved by #678
